test/regression/carousel.hermione.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable babel/no-invalid-this */
+'use strict';
+
+describe('React Carousel', () => {
+  it('should swipe slide', async function() {
```

Comment:
```suggestion
  it('swipes slide', async function() {
```

Reply:
I see inconsistency as some test messages start with `should` while others with `Should` so maybe we should skip the `should` as I prefer